vagrant/bootstrap.sh

```diff
@@ -10,4 +10,4 @@ apt-get install -y \
 	mercurial \
 	python3-pip \
 	srecord \
-	unzip \
+	unzip
```

Comment:
Please leave the trailing \ so that the diff is clean when we add items after unzip. There is a newline after line 13 so this should work fine; if not, add a newline instead of removing the `\`.

Reply:
Done